starknet_py/net/account/account_client.py

```diff
@@ -477,27 +477,25 @@ async def execute(
     async def deploy(self, transaction: Deploy) -> DeployTransactionResponse:
         return await self.client.deploy(transaction=transaction)
 
-    async def deploy_contract(
+    async def deploy_through_udc(
```

Comment:
I'm not quite sure if that's a good name. I'd prefer if we avoided abbreviations in method names.